### PR TITLE
Updating Language/Language SDK with new /entities endpoint

### PR DESCRIPTION
**Pull request by assafi**
## Description

- Update of existing TA SDK with new /entities endpoint and additional Azure regions. 
- The SDK will be deprecated and succeeded by `Microsoft.Azure.CognitiveServices.Language.TextAnalytics` NuGet. See [new SDK PR](https://github.com/Azure/azure-sdk-for-net/pull/4285).

[Swagger spec PR](https://github.com/Azure/azure-rest-api-specs/pull/2985)

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
@assafi  The `PackageReleaseNotes` here must be updated once the new packages are published

**Comment by assafi** [comment]
@dsgouda The `PackageReleaseNotes` have been updated. What additional info would you like to see?

**Comment by dsgouda** [comment]
@assafi Once you publish Language.TextAnalytics package, note the link to that package and add it to the release notes here,

File: src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/Microsoft.Azure.CognitiveServices.Language.csproj [hunk 1]
```diff
@@ -6,14 +6,13 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Language</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Language APIs.</Description>
-    <VersionPrefix>1.0.1-preview</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/Microsoft.Azure.CognitiveServices.Language.csproj]: Replace `VersionPrefix` with `Version`

File: src/SDKs/CognitiveServices/dataPlane/Language/Language/Language.Tests/Microsoft.Azure.CognitiveServices.Language.Tests.csproj [hunk 2]
```diff
@@ -3,11 +3,11 @@
   <PropertyGroup>
     <Description>Microsoft.Azure.CognitiveServices.Language.Tests Class Library</Description>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Language.Tests</AssemblyName>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/Language/Language.Tests/Microsoft.Azure.CognitiveServices.Language.Tests.csproj]: no changes required to this file

**Review by dsgouda — CHANGES_REQUESTED** [review]
Are you able to generate the code using generate.cmd? please do if you can and commit all changes

**Comment by assafi** [comment]
For this SDK update I am not able to use the generate.ps1 since the namespace has changed. However, I've used the generated code from the new SDK and copied it over since it's exactly the same except for the project path. I just copied these changes over.

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will pass on CIs passing

**Review by dsgouda — APPROVED** [review]
LGTM will merge on CIs passing

**Pull request merged**